docs/index.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Welcome to Glade Forms!
+description: Versatile Forms framework
+---
+
+Developed with 💚 by [netglade][netglade_link]
+
+[![ci][ci_badge]][ci_badge_link]
+[![glade_forms][glade_forms_pub_badge]][glade_forms_pub_badge_link]
+[![license: MIT][license_badge]][license_badge_link]
+[![style: netglade analysis][style_badge]][style_badge_link]
+[![Discord][discord_badge]][discord_badge_link]
```

Comment:
Place to one line using HTML similarly to 

https://docs.jaspr.site
https://raw.githubusercontent.com/schultek/jaspr/refs/heads/main/docs/index.mdx